Tests/Parser/Client/fixtures/mobile_app.yml

```diff
@@ -2069,3 +2069,15 @@
     type: mobile app
     name: Zalo
     version: ""
+-
+  user_agent: Mozilla/5.0 (Macintosh; U; Intel Mac OS X 10.6; en-US; rv:1.9.2.3) Gecko/20100402 Prism/1.0b4
+  client:
+    type: mobile app
```

Comment:
From description of the product, it looks like it was more like a library allowing to integrate Gecko engine. Probably would make more sense as browser with Gecko engine.